tests/SerializationFormatSpec.hs

```diff
@@ -83,7 +83,7 @@ jsonExamples =
   , example "NonEmpty" "[1,2,3]"  (1 :| [2, 3] :: NonEmpty Int)
   , example "Seq" "[1,2,3]"  (Seq.fromList [1, 2, 3] ::  Seq.Seq Int)
   , example "DList" "[1,2,3]"  (DList.fromList [1, 2, 3] :: DList.DList Int)
-  , example "()" "[]"  ()
+  , Example "()" ["[]", "{}"]  () ()
```

Comment:
The extra case should be added to `jsonDecodingExamples`. This example (which is `ndExample`) tells that there would be a system that `()` could encode as `{}`, which isn't true.

Reply:
I believe this issue is now fixed.